src/admin/adminSlice.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Why is this required?

Reply:
The `no-param-reassign` rule in our eslint config has the option `{props: true}` (from airbnb) which forbids mutating properties of parameters to reduce side effects. However, since `redux-toolkit` uses immer for `createSlice`, it's more idiomatic to mutate the `state` argument when defining reducers since it'll be equivalent to copying the parameter and is generally easier to read as well. I still think it's a good rule, so I've just disabled it in all slice files since createSlice will usually mutate the state parameter.

`redux-toolkit` has a [discussion](https://github.com/reduxjs/redux-toolkit/issues/521) on this which tldr: airbnb is too restrictive, just use `{props: false}` which is the eslint default anyway. 

Thoughts?